packages/forma-36-react-components/src/components/Workbench/Workbench.css

```diff
@@ -0,0 +1,99 @@
+@import 'settings/colors';
+@import 'settings/typography';
+@import 'settings/transitions';
+@import 'settings/dimensions';
+@import 'settings/shadows';
+@import 'settings/content-width';
+
+.Workbench {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  z-index: 0;
+  overflow-x: hidden;
+}
+
+.Workbench__header {
+  box-sizing: border-box;
+  position: relative;
+  width: 100%;
+  background-color: var(--color-element-lightest);
+  z-index: 6;
+  height: 70px;
+  display: flex;
+  flex-direction: row;
+  flex-shrink: 0;
+  align-items: center;
+  padding: var(--spacing-l);
+  border-bottom: 1px solid var(--color-element-darkest);
+  box-shadow: var(--box-shadow-default);
+}
+
+.Workbench__header-icon {
+  margin-right: var(--spacing-m);
+}
+
+.Workbench__header-title {
+  flex-grow: 1;
+}
+
+.Workbench__header-actions {
+  margin-left: var(--spacing-l);
+  display: flex;
+  flex-direction: row;
+}
+
+.Workbench__content-wrapper {
+  display: flex;
+  flex-direction: row;
+  overflow-y: auto;
+  position: relative;
+  height: calc(100% - 71px);
+}
+
+.Workbench__content {
+  flex-grow: 1;
+  overflow-x: hidden;
+  overflow-y: auto;
+  padding: var(--spacing-l);
+}
+
+.Workbench__content-inner {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.Workbench__content-inner--default {
+  max-width: var(--content-width-default);
+}
+
+.Workbench__content-inner--text {
+  max-width: var(--content-width-text);
+}
+
+.Workbench__content-inner--full {
+  max-width: var(--content-width-full);
+}
+
+.Workbench__sidebar {
+  padding: var(--spacing-l);
+  text-align: left;
+  overflow-x: hidden;
+  overflow-y: auto;
+  z-index: 0;
+  background-color: var(--color-element-lightest);
+}
+
+.Workbench__sidebar--left {
```

Comment:
Wouldn't it be better if the left and right sidebars were separate components which you can choose to use? To make things really customisable we could let people opt to use something that doesn't have this width constraint.

Reply:
Yeah, probably it makes sense to give possibility to change predefined width. @wichniowski wdyt?

Reply:
I guess it's actually required to make the sidebar a separate component seeing as they already use different widths? I'm a little concerned this could add more inconsistency though.

Reply:
In my idea, you would always have to use the `<WorkbenchSidebar>` component, even for custom ones, but not necessarily the render props for left and right.

You can still opt in for the predefined widths by _optionally_ setting the `position` prop (which could maybe be renamed to smth like `sidebarType`)